src/styles/typography.js

```diff
@@ -76,80 +34,3 @@ export const heading = (theme, level = 0) => ({
   lineHeight: theme.msRatio,
   position: "relative"
 });
-
-// Block element styles
-
-export const textBlock = (theme, selector = "p", level = 0) => ({
-  [selector]: {
-    ...text(theme, level),
-    flexBasis: "100%",
-    margin: `16px 0 0 0`
-  },
-  ...inlineElements(theme, `${selector} >`)
-});
-
-export const headingBlock = (theme, selector = "h1", level = 0) => ({
-  [selector]: {
-    ...heading(theme, level),
-    flexBasis: "100%",
-    margin: `48px 0 0 0`
-  },
-  ...adjacent(["blockquote", "h1", "h2", "h3", "h4", "h5", "h6"], selector, {
-    margin: `16px 0 0 0`
-  }),
-  ...inlineElements(theme, `${selector} >`)
-});
-
-export const unorderedList = (theme, selector = "ul", level = 0, depth = 0) => {
```

Comment:
I can't see how nested lists are handled in the new `Content/Markdown.js` – does it still work? I think it should …

Reply:
Each component "owns" its styling now. We don't need to select nested lists from the top like `.cg-Page > ul > li > ul` anymore.

List items just remove the top and bottom margin of their first/last child respectively.